todo-angular/src/app/todo/todolist/todolist.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { TaskModel, TodoModel } from 'src/Models/TodoModel';
+import { TODO_LIST_ID } from '../todo-routing.module';
+import { TodoService } from './../todo.service';
+
+@Component({
+  selector: 'app-todolist',
+  templateUrl: './todolist.component.html',
+  styleUrls: ['./todolist.component.scss'],
+})
+export class TodolistComponent implements OnInit {
+
+  todoListTasks: TaskModel[] = [];
+
+  constructor(private readonly todoService: TodoService, private readonly route: ActivatedRoute) {}
+
+  ngOnInit(): void {
+    const todoListId = this.route.snapshot.paramMap.get(TODO_LIST_ID); // Læser parameteren "TODO_LIST_ID" fra URL'en (sat i todo-routing module)
+    this.getSpecificTodos(todoListId);
+  }
+
+  getSpecificTodos(id: string | null) {
+    this.todoService.getSpecificTodo(id).subscribe({
+      next: (tasks) => {
+        this.todoListTasks = tasks;
+      }
+    });
+  }
+
+  onCheckboxChanged(task: TaskModel) {
+    this.todoService.updateTask(task.todoTaskId, task).subscribe();
```

Comment:
In the subscribe next handler, you can update the todo with the correct id the list so that it is completed

Reply:
Do you still want to do this, when it is a put call, that returns nothing (204 No Content)?

Reply:
Hmm If that's the case, I want you to switch the completed property on the task in the subscribe :)